scripts/linters/js_ts_linter.py

```diff
@@ -220,6 +228,57 @@ def _compile_ts_file(self, filepath, dir_path):
             dir_path, os.path.basename(filepath).replace('.ts', '.js'))
         return compiled_js_filepath
 
+    def _check_any_type(self):
+        """Checks if the type of any variable is declared as any
+        in typescript files.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting any type check')
+            python_utils.PRINT('----------------------------------------')
+
+        # First pattern is used to match cases like these : any
+        # Second pattern is used when the character of last line was :
+        # So, we know that this line begins with a type of variable
+        patterns_to_match = [r':\ *any', r'^\ *any']
+
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+            summary_messages = []
+
+            for file_path in self.all_filepaths:
+                if file_path in FILES_EXCLUDED_FROM_ANY_TYPE_CHECK:
+                    continue
+
+                file_content = FILE_CACHE.read(file_path)
+                starts_with_type = False
+
+                for line_number, line in enumerate(file_content.split('\n')):
+                    if ((starts_with_type and re.findall(
+                            patterns_to_match[1], line)) or re.findall(
+                                patterns_to_match[0], line)):
```

Comment:
Rewrite this if into two cases maybe, so it is more clear to read.

Reply:
Done